data/virt_autotest/guest_params_xml_files/sles_15_sp3_64_xen_hvm_uefi_with_power_management_x86_64.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp3</guest_version>
+    <guest_version_major>15</guest_version_major>
+    <guest_version_minor>3</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>xen</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel></guest_cpumodel>
+    <guest_metadata></guest_metadata>
+    <guest_xpath></guest_xpath>
+    <guest_installation_automation>autoyast</guest_installation_automation>
+    <guest_installation_autoyast_file>sles_15_plus_xen_hvm_guest_graphical_x86_64.xml</guest_installation_autoyast_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1</guest_installation_extra_args>
+    <guest_installation_wait></guest_installation_wait>
+    <guest_installation_method_others></guest_installation_method_others>
+    <guest_installation_media>http://openqa.suse.de/assets/repo/SLE-15-SP3-Full-x86_64-Build12345-Media1</guest_installation_media>
```

Comment:
As the same, we will use this `guest_installation_media` as http://openqa.suse.de/assets/repo/SLE-15-SP3-Full-x86_64-Build12345-Media1 all time, right?

Reply:
This is not fixed. You can modify it or even create new profiles with desired parameters with desired media. There is no restriction.

Reply:
yep, gotcha.